tests/integration/src/test/resources/org/mule/test/integration/notifications/modules/module-simple-proxy.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module name="module-simple-proxy"
+        minMuleVersion="4.0.0"
+
```

Comment:
it's on purpose, as it's meant to be there to be sure that line breaks are properly read by the component model